options.go

```diff
@@ -164,6 +165,7 @@ func NewOptions() *Options {
 		SetXAuthRequest:                  false,
 		SkipAuthPreflight:                false,
 		PassBasicAuth:                    true,
+		SetBasicAuth:                     true,
```

Comment:
This should default to `false` otherwise this will always reflect peoples credentials back at them